bundle/next/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -75,12 +99,127 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/eclipse-che/che-operator
     support: Eclipse Foundation
-  name: eclipse-che-preview-openshift.v7.46.0-440.next
+  name: eclipse-che-preview-openshift.v7.47.0-520.next
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+      - description: 'The `CheCluster` custom resource allows defining and managing
+          Eclipse Che server installation. Based on these settings, the  Operator
+          automatically creates and maintains several ConfigMaps: `che`, `plugin-registry`,
+          `devfile-registry` that will contain the appropriate environment variables
+          of the various components of the installation. These generated ConfigMaps
+          must NOT be updated manually.'
+        displayName: Eclipse Che instance Specification
+        kind: CheCluster
+        name: checlusters.org.eclipse.che
+        resources:
+          - kind: ClusterRole
+            name: ''
+            version: v1
+          - kind: ClusterRoleBinding
+            name: ''
+            version: v1
+          - kind: ConfigMap
+            name: ''
+            version: v1
+          - kind: Deployment
+            name: ''
+            version: apps/v1
+          - kind: Ingress
+            name: ''
+            version: v1
+          - kind: Role
+            name: ''
+            version: v1
+          - kind: RoleBinding
+            name: ''
+            version: v1
+          - kind: Route
+            name: ''
+            version: v1
+          - kind: Secret
+            name: ''
+            version: v1
+          - kind: Service
+            name: ''
+            version: v1
+        specDescriptors:
+          - description: Container registry configuration.
```

Comment:
Can we have it displayed in this order:
- workspaces
- serverComponents
- ingress
- containerRegistry

Reply:
Yes we can.